sklearn/model_selection/_validation.py

```diff
@@ -610,19 +613,37 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     return result
 
 
-def _score(estimator, X_test, y_test, scorer):
+def _score(estimator, X_test, y_test, scorer, error_score=np.nan):
```

Comment:
Giving this a default value means that we risk missing usages that do not specify the behaviour. Leave it as
```suggestion
def _score(estimator, X_test, y_test, scorer, error_score):
```